### ISSUE TITLE
Implement a tool to plot the EIC given a mass using XCMS

### ISSUE THREAD

**Issue by hechth** [RECETOX/galaxytools#498]
The tool should visualize the extracted ion chromatogram (EIC) given a single mzml file (in a first step), the MZ value at which to extract the slice and the tolerance to use for extraction.

The documentation for the R packages which should be used for these operations is here: https://jorainer.github.io/xcmsTutorials/articles/xcms-preprocessing.html#data-visualization
This also includes a tutorial of how to call the functions as well as example outputs.

XCMS is already as a package in bioconda via bioconductor, same as `Spectra` and `MetaboCoreUtils`, they can therefore be specified as a requirement in the galaxy tool if they are needed.

In a next step, it would be great if the tool also worked on a collection and it would be possible to add a legend with the sample names and different colors for the samples. This could then also be extended towards the formats that mzR/XCMS/Spectra can handle, meaning `mzData`, `mzxml` and `raw`.

Another option for extension would be to implement something similar like is done [here](https://bioconductor.org/packages/release/bioc/vignettes/rawrr/inst/doc/rawrr.html#32_Use_Case_II_-_iRT_Regression_for_System_Suitability_Monitoring) with rawRR to plot multiple EICs of a single sample in a single plot.

TLDR

- [x] tool which runs on mzml and gives an EIC as PNG
- [ ] rename the output to `eic at <mz> of <dataset id>`
- [ ] update the help text
- [ ] add reference to jo rainers tutorial (once on zenodo)
- [ ] extend the tool to mzXML and mzData
- [ ] implement galaxy tool test for mzML
